### ISSUE TITLE
Talk more about the difference between score and estimated probability

### ISSUE THREAD

**Issue by uriahf** [uriahf/performance_evaluation_of_prediction_models#2]
According to Ofer it is not clear enough when I talk about score and when I talk about estimated probability, maybe because I'm coming from different domain.

**Comment by oferisak** [comment]
Yes, it has to be clear that a score is a score and not a probability until proven otherwise.. perhaps discuss the conditions in which you can say a score corresponds to probability for example - 
- the training sample should correspond to the population the model will run on
- the model is well calibrated
